### PR TITLE
rename namespace field to serviceVendor.

### PR DESCRIPTION
**Pull request by Alice1319**
fixes: #2188

### CODE REVIEW

File: modules/security/src/main/java/org/eclipse/xpanse/modules/security/common/CurrentUserInfo.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class CurrentUserInfo {
 
     private Map<String, String> metadata;
 
-    private String namespace;
+    private String serviceVendor;
```
Comment (baixinsui) [inline modules/security/src/main/java/org/eclipse/xpanse/modules/security/common/CurrentUserInfo.java:25]: This should be 'isv'

**Review by baixinsui — COMMENTED** [review]
All keys 'namespace' should be updated to 'isv' in all jwt_xxx.json as below:
image

File: modules/security/src/main/java/org/eclipse/xpanse/modules/security/common/CurrentUserInfo.java (hunk 1 above)
Comment (Alice1319) [reply modules/security/src/main/java/org/eclipse/xpanse/modules/security/common/CurrentUserInfo.java]: updated

File: modules/api/src/main/java/org/eclipse/xpanse/api/config/ServiceTemplateEntityConverter.java [hunk 2]
```diff
@@ -37,10 +37,11 @@ public static ServiceTemplateDetailVo convertToServiceTemplateDetailVo(
             serviceTemplateDetailVo.setServiceTemplateId(serviceTemplateEntity.getId());
             serviceTemplateDetailVo.setIcon(serviceTemplateEntity.getOcl().getIcon());
             serviceTemplateDetailVo.setDescription(serviceTemplateEntity.getOcl().getDescription());
-            if (StringUtils.isNotEmpty(serviceTemplateEntity.getNamespace())) {
-                serviceTemplateDetailVo.setNamespace(serviceTemplateEntity.getNamespace());
+            if (StringUtils.isNotEmpty(serviceTemplateEntity.getServiceVendor())) {
+                serviceTemplateDetailVo.setServiceVendor(serviceTemplateEntity.getServiceVendor());
             } else {
```
Comment (baixinsui) [inline modules/api/src/main/java/org/eclipse/xpanse/api/config/ServiceTemplateEntityConverter.java:42]: The if... else... code is unnecessary, the value already has been set by called BeanUtils.copyProperties(serviceTemplateEntity, serviceTemplateDetailVo) .

**Comment by baixinsui** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Comment by luoping-12345** [comment]
LGTM

**Pull request merged**